man/systemd-analyze.xml

```diff
@@ -770,6 +770,18 @@ Service b@0.service not loaded, b.socket cannot be started.
         operate on files inside the specified image path <replaceable>PATH</replaceable>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--offline=<replaceable>BOOL</replaceable></option></term>
+
+        <listitem><para>With <command>security</command>, perform an offline security review
+        of the specified unit file(s), i.e. does not have to rely on PID 1 to acquire security
+        information for the files like the <command>security</command> verb when used by itself does.
+        This means that <option>--offline</option> can be used with <option>--root</option> and
```

Comment:
command line switches that expact values we typically suffix with `=` in our docs. i.e. type this as `--offline=`, `--root=`, `--image=`, `--threshold=` here